unit_tests/test_packages.py

```diff
@@ -20,6 +19,9 @@ def setUp(self):
         super(PackagesTest, self).setUp()
         self.apt = AptStub()
         self.packages = Packages(apt=self.apt)
+        keyfile = "jenkins.io.key"
+        os.symlink(os.path.join(os.getcwd(), keyfile),
+                   os.path.join(hookenv.charm_dir(), keyfile))
```

Comment:
Ah interesting. Mind filing an issue against [charm-test](https://github.com/freeekanayaka/charm-test)? Then please write an "XXX" comment just above this code describing what you said here and put a reference to the issue that you filed.

I'll see how this could be streamlined in charm-test and then fix this.